src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-editor.component.ts

```diff
@@ -500,6 +502,6 @@ export class ModelingAssessmentEditorComponent implements OnInit {
      * and instead set the score boundaries on the server.
      */
     calculateTotalScore() {
-        this.totalScore = (this.feedback || []).reduce((totalScore, feedback) => totalScore + feedback.credits!, 0);
+        this.totalScore = this.structuredGradingCriterionService.computeTotalScore(this.feedback);
```

Comment:
Just a small note: the `maxCountInScore` feature is not integrated in Apollon yet so calculating the score using `structuredGradingCriterionService.computeTotalScore(this.feedback)` in modeling-assessment-editor will not change the score calculation (it will skip all maxCountInScore related conditions). Have you tested assessing modeling exercises after these changes? (everything should work as expected but just in case ^^')